packages/admin/cms-admin/src/contentScope/ContentScopeSelect.tsx

```diff
@@ -84,7 +84,12 @@ export function ContentScopeSelect<Value extends ContentScopeInterface = Content
     }
 
     const selectedOption = options.find((option) => {
-        return Object.keys(option).every((key) => value[key] === option[key].value);
+        const dimensions = new Set<string>();
+        for (const dimension of [...Object.keys(value), ...Object.keys(option)]) {
+            dimensions.add(dimension);
+        }
+
+        return Array.from(dimensions).every((key) => value[key] === option[key]?.value);
```

Comment:
I'm not sure if we'll understand this code in 6 months 😅  Wouldn't `Object.keys(value).every((key) => value[key] === option[key].value);` work as well?

Reply:
No because in that case the value `{ country: "at" }` would also match the option `{ country: "at", company: "A" }` because you never check for `company` -> Which one matches depends on the order of the options.


(If the value is `{ country: "at", company: undefined }` it should work, but I don't want to rely on that)